src/test/java/org/folio/rest/impl/AutomatedPatronBlocksAPITest.java

```diff
@@ -110,240 +113,273 @@ public void shouldReturnNoBlocksWhenNoLimitsExistForPatronGroup() {
   }
 
   private void validateMaxNumberOfItemsChargedOutBlockResponse(int openLoansSizeDelta,
-    boolean singleLimit, boolean blockShouldExist) {
+    boolean singleLimit) {
 
     final Condition condition = MAX_NUMBER_OF_ITEMS_CHARGED_OUT;
+    updateCondition(condition, expectBlockBorrowing, expectBlockRenewals, expectBlockRequests);
```

Comment:
Can we set all these to `true` by default before all tests?

Reply:
Done